onnx/backend/test.py

```diff
@@ -122,12 +123,12 @@ def _prepare_model(self, model_name):
             os.makedirs(model_dir)
             url = 'https://s3.amazonaws.com/download.onnx/models/{}.tar.gz'.format(
                 model_name)
-            download_file = tempfile.NamedTemporaryFile(delete=True)
-            print('Start downloading model {} from {}'.format(model_name, url))
-            urlretrieve(url, download_file.name)
-            print('Done')
-            with tarfile.open(download_file.name) as t:
-                t.extractall(models_dir)
+            with tempfile.NamedTemporaryFile(delete=True) as download_file:
```

Comment:
It surprises me that a linter can find this out

Reply:
Yup, quite helpful.